src/shared/components/boardCard/boardCard.css

```diff
@@ -63,3 +63,14 @@
   align-self: left;
   padding: 1rem 0 0 0;
 }
+
+@media screen and (max-width: 850px) {
```

Comment:
This won't deactivate the animation.

You'll want to set `transition: none` on the classes and states involved (:hover, :active, etc).

Let's do it on devices under 450px in width instead of 850, also. The animations should work okay on tablets.